keras_tuner/engine/oracle.py

```diff
@@ -236,11 +236,10 @@ def update_trial(self, trial_id, metrics, step=0):
     def end_trial(self, trial_id, status="COMPLETED"):
         """Record the measured objective for a set of parameter values.
 
-        Arguments:
-            trial_id: String. Unique id for this trial.
-            status: String, one of "COMPLETED", "INVALID". A status of
-                "INVALID" means a trial has crashed or been deemed
-                infeasible.
+        Args:
+            trial_id: A string, the unique ID for this trial.
+            status: A string, one of "COMPLETED", "INVALID". A status of
```

Comment:
When mentioning strings, use both backticks (because it's code) and quotes (since it's a string within the code block), e.g. `"COMPLETED"`